tsconfig.json

```diff
@@ -1,11 +1,10 @@
 {
   "compilerOptions": {
-    "target": "ESNext",
-    "module": "ESNext",
-    "moduleResolution": "Node",
-    "strict": false,
+    "target": "ES2020",
+    "module": "CommonJS",
```

Comment:
why do we need ES2020 and commonJS?

Reply:
this fixes all the replace imports stuff, not entirely sure but it works on my end